automation/e2e-upgrade-functests/run.sh

```diff
@@ -18,7 +18,7 @@ SOURCE_DIR=$(dirname "$0")
 NAMESPACE=${1:-kubevirt}
 
 RELEASE_BRANCH=${CI_BRANCH}
-if [[ -z ${RELEASE_BRANCH} ]] || [[ ${RELEASE_BRANCH} == "master" ]]
+if [[ -z ${RELEASE_BRANCH} ]] || [[ ${RELEASE_BRANCH} == "main" ]] || [[ ${RELEASE_BRANCH} == "master" ]]
```

Comment:
Can this still occur?

Reply:
We need both master and main to pass tests in `openshift/release` repository, after it gets merged I will remove it in a follow up.